app/views/login.html

```diff
@@ -145,31 +145,28 @@
     <script type="application/javascript">
     document.write("<script src='http://" + (location.host || "localhost").split(":")[0] + ":35729/livereload.js'></" + "script>");
 
-    function areCookiesEnabled() {
-        var cookieEnabled = navigator.cookieEnabled;
+    const areCookiesEnabled = () => {
+        const cookieEnabled = navigator.cookieEnabled;
 
         // When cookieEnabled flag is present and false then cookies are disabled.
-        if (cookieEnabled === false) {
-            return false;
-        }
+        if (cookieEnabled === false) return false;
```

Comment:
Maybe we can update this to `if (!cookieEnabled) return false`

Reply:
Yeah, good suggestion, I didn't notice that, thanks !